mlflow/pyfunc/__init__.py

```diff
@@ -1108,6 +1108,7 @@ def spark_udf(
     result_type=None,
     env_manager=_EnvManager.LOCAL,
     params: Optional[Dict[str, Any]] = None,
+    env_vars=None,
```

Comment:
The argument name is not very clear, what about renaming it to `extra_udf_envs` ?

Reply:
I'll raise a separate PR about this change as it's irrelevant to langchain